src/GraphQL/Types/TimeSpanSecondsGraphType.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using GraphQL.Language.AST;
+
+namespace GraphQL.Types
+{
+    public class TimeSpanSecondsGraphType : ScalarGraphType
+    {
+        public TimeSpanSecondsGraphType()
+        {
+            Name = "Seconds";
+            Description =
+                "The `Seconds` scalar type represents a period of time represented as the total number of seconds.";
+        }
+
+        public override object Serialize(object value)
+        {
+            return ParseValue(value);
```

Comment:
I think this need to be changed to similar to the `Milliseconds` type?  Checking for `TimeSpan` and then choosing `TimeSpan.TotalSeconds`?